src/main/java/io/minepkg/testutils/RuleBookGUI.java

```diff
@@ -146,16 +147,17 @@ public RuleBookGUI(World w, PlayerEntity player) {
 
     setRootPanel(root);
     root.setSize(100, 100);
+    root.setInsets(Insets.ROOT_PANEL);
```

Comment:
This was necessary to fix this:
![oof](https://user-images.githubusercontent.com/8464472/125671448-fbd79e56-4456-45e4-9416-62a24d9b25d0.png)